services/provider/server/server.go

```diff
@@ -666,7 +666,42 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					Kind: "VolumeGroupSnapshotClass",
 					Data: mustMarshal(map[string]string{
 						"csi.storage.k8s.io/group-snapshotter-secret-name": provisionerSecretName,
-					})})
+					})},
+				&pb.ExternalResource{
+					Name: "ceph-rbd",
+					Kind: "VolumeReplicationClass",
+					Data: mustMarshal(map[string]string{
+						"replication.storage.openshift.io/replication-secret-name": provisionerSecretName,
+						"replication.storage.openshift.io/is-default-class":        "true",
+					}),
+				},
+				&pb.ExternalResource{
+					Name: "ceph-rbd-image-flattening",
+					Kind: "VolumeReplicationClass",
+					Data: mustMarshal(map[string]string{
+						"replication.storage.openshift.io/replication-secret-name": provisionerSecretName,
+						"flattenMode": "force",
+						"replication.storage.openshift.io/flatten-mode": "force",
+					}),
+				},
+				&pb.ExternalResource{
+					Name: "ceph-rbd",
+					Kind: "VolumeGroupReplicationClass",
+					Data: mustMarshal(map[string]string{
+						"replication.storage.openshift.io/group-replication-secret-name": provisionerSecretName,
+						"replication.storage.openshift.io/is-default-class":              "true",
+					}),
+				},
+				&pb.ExternalResource{
+					Name: "ceph-rbd-image-flattening",
```

Comment:
```suggestion
					Name: "ceph-rbd-flatten",
```

Please make sure everything matches with https://github.com/red-hat-storage/odf-multicluster-orchestrator/blob/b3df88c23b06d80515f27a36f656296e2d413681/controllers/drpolicy_controller.go#L212

Reply:
I notice the name's suffix is a hashed scheduling interval. Since I am watching the new DRClusterConfig resource (that provides the sched intervals) from the client op, I'd need to override this field there. But I guess it doesn't matter on your end because all you care about is that the client op sends the correct name.

Reply:
The naming will be handled on consumer's end. Could you please resolve? @Rakshith-R

Reply:
I think i already mentioned it looks good to me here 
https://github.com/red-hat-storage/ocs-operator/pull/2727#pullrequestreview-2224881050